src/components/component/reviews/ReviewCard.jsx

```
@@ -0,0 +1,47 @@
+            }
+          })}
+        </div>
+      </div>
+      <p className="review-content">{review.contents}</p>
+      <p className="review-createdAt">{review.createdAt}</p>
+      <div className="review-actions">
+        <button
+          className={`btn-icon ${review.liked ? "active" : ""}`}
+          onClick={() => handleLike(review.id, review.liked)}
+          disabled={likedloading}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)